tests/migration/migration.go

```diff
@@ -316,10 +316,12 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 			// execute a migration, wait for finalized state
 			By("Starting the Migration for iteration")
+			sourcePod := tests.GetRunningPodByVirtualMachineInstance(vmi, testsuite.GetTestNamespace(vmi))
 			migration := libmigration.New(vmi.Name, vmi.Namespace)
 			migration = libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(virtClient, migration)
 			By("Checking VMI, confirm migration state")
-			libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
+			vmi = libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
+			Expect(vmi.Status.MigrationState.SourcePod).To(Equal(sourcePod.Name))
```

Comment:
ExpectWithOffset would be better in that case

Reply:
The rest of the function doesn't use `WithOffset`, and I would argue that it's a good thing.
`WithOffset` effectively obfuscates which line of the function tripped. and this is a massive function called once by 2 different tests.
The code for the first test that uses the function is 2 lines long, and the second one has just 1 line: the call to this function!
So, in this case, moving the failure up one level effectively makes the line number meaningless.
WDYT?

Reply:
Sounds good :)